test/e2e/common.go

```diff
@@ -7,63 +7,94 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// newPod returns a new Pod object.
-func newPod(namespace string, name string, containerName string, runtimeclass string) *corev1.Pod {
-	return &corev1.Pod{
-		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: namespace},
-		Spec: corev1.PodSpec{
-			Containers:       []corev1.Container{{Name: containerName, Image: "nginx"}},
-			DNSPolicy:        "ClusterFirst",
-			RestartPolicy:    "Never",
-			RuntimeClassName: &runtimeclass,
-		},
+type RuntimeClass string
+
+const (
+	RuntimeClassKata       RuntimeClass = "kata"
+	RuntimeClassKataCLH    RuntimeClass = "kata-clh"
+	RuntimeClassQemu       RuntimeClass = "kata-qeum"
```

Comment:
s/kata-qeum/kata-qemu

Reply:
@mattarnoatibm - I think you'll need to squash/fixup the commits to clean up the commit history, sorry I missed that yesterday